addons/purchase/wizard/bill_to_po_wizard.py

```diff
@@ -0,0 +1,82 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models, Command, _
+from odoo.exceptions import UserError
+
+
+class BillToPO(models.TransientModel):
+    _name = 'bill.to.po.wizard'
+    _description = 'Bill to Purchase Order'
+
+    display_add_button = fields.Boolean()
+    purchase_order_id = fields.Many2one(comodel_name='purchase.order')
+    partner_id = fields.Many2one(comodel_name='res.partner')
+
+    # @api.depends_context('active_ids')
+    # def _compute_aml_ids(self):
+    #     print('_compute_aml_ids')
+    #     aml_ids = [abs(record_id) for record_id in self.env.context.get('active_ids') if record_id < 0]
+    #     for wizard in self:
+    #         wizard.aml_ids = self.env['account.move.line'].browse(aml_ids)
+
+    def action_add_to_po(self):
+        aml_ids = [abs(record_id) for record_id in self.env.context.get('active_ids') if record_id < 0]
+        lines_to_add = self.env['account.move.line'].browse(aml_ids).filtered(lambda l: l.product_id)
+        if not lines_to_add:
+            raise UserError(_("There are no products to add to the Purchase Order. Are these Down Payments?"))
```

Comment:
The Add to PO button is hidden when the account move lines don't contain products.
Is this necessary?
I think we can keep it as an additional check since the "missing product" constraint will cause a stack trace